packages/verified-fetch/test/fixtures/dns-answer-fake.ts

```diff
@@ -0,0 +1,13 @@
+import { stubInterface } from 'sinon-ts'
+import type { DNSResponse } from '@multiformats/dns'
+
+export function answerFake (data: string, TTL: number, name: string, type: number): DNSResponse {
```

Comment:
i believe this is in the https://github.com/ipfs/helia-verified-fetch/pull/19 PR, and i needed it here.